### DISCUSSION TITLE
UpdateSourceTrigger=LostFocus

### DISCUSSION THREAD

**Discussion by qfmee** [AvaloniaUI/Avalonia#13739] [General]
We hope that Textbox will add UpdateSourceTrigger=LostFocus in the next version, which is very, very important to us. Thank you to the Avalonia team！

**Comment by thevortexcloud** [comment]
There is an issue [here](https://github.com/AvaloniaUI/Avalonia/issues/3754) to track `UpdateSourceTrigger` and also mentions some possible workarounds. But if you elaborate more on why you need it someone else may have other ideas.

**Reply by thevortexcloud** [reply]
Apparently it's being added now.

https://github.com/AvaloniaUI/Avalonia/commit/bee98ad9479a2c27dfd68dc38b09b01882b1c762#diff-ccfdf6768edfce7a5cf7bb4a103e15abeeb53ba684bf457df6d52d3f61a9cdb2

**Reply by qfmee** [reply]
thanks
